OSDOCS-17900 created zstream RNs for 4-19-22 (#105065)
openshift/openshift-docs: modules/zstream-4-19-22-fixed-issues.adoc

* Before this update, a bug prevented the `ValidatingAdmissionPolicy` resource from applying to certain {product-title} API resources, such as `BuildConfig` and `DeploymentConfig`. This meant that custom admission policies were not enforced on these specific resources, potentially allowing configurations that did not meet organizational standards to be created or updated. With this release, the validation logic has been corrected to ensure that the`ValidatingAdmissionPolicy` resource now correctly identifies and applies to all intended {product-title} resources. As a result, users can consistently enforce policies across their entire cluster, including the `BuildConfig` and `DeploymentConfig` resources. (link:https://issues.redhat.com/browse/OCPBUGS-66920[OCPBUGS-66920])

> * Before this update, changes in RHCOS image layering increased space usage on the rendezvous node's ephemeral temporary file system to approximately 9.4GB during cluster bootstrapping. Because the ephemeral temporary file system is capped at 50% of available RAM, installation would fail on hosts with less than 19GiB of memory due to insufficient space for container images. With this release, the additional data has been moved to a separate temporary file system. As a result, any rendezvous host meeting the minimum RAM requirement for a control plane node (16GB) now has sufficient capacity to successfully bootstrap a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-66999[OCPBUGS-66999])

Reviewer: ```suggestion
* Before this update, changes in {image-mode-os-lower} increased space usage on the rendezvous node's ephemeral temporary file system to approximately 9.4GB during cluster bootstrapping. Because the ephemeral temporary file system is capped at 50% of available RAM, installation would fail on hosts with less than 19GiB of memory due to insufficient space for container images. With this release, the additional data has been moved to a separate temporary file system. As a result, any rendezvous host meeting the minimum RAM requirement for a control plane node (16GB) now has sufficient capacity to successfully bootstrap a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-66999[OCPBUGS-66999])
```
I am assuming that RHCOS image layering is the same as Image mode for OpenShift (the name changed in 4.19)

* Before this update, the deployment agent was unable to find a valid IP address when the Ironic API was running on a non-standard port because the reachability test only used the hostname. This limitation resulted in a `LookupAgentIPError`, preventing successful connections and halting the deployment process in environments with custom port configurations. With this release, the reachability test has been updated to use the full API URL, including the specific port number, instead of relying solely on the hostname. As a result, the agent can now consistently identify the correct IP address and connect to Ironic APIs on any port, ensuring smoother deployments and eliminating the previous lookup errors. (link:https://issues.redhat.com/browse/OCPBUGS-67304[OCPBUGS-67304])

> * Before this update, [product-title} 4.16 and later versions failed to respect the timeout `http-keep-alive` setting due to a known upstream High HAProxy bug, preventing users from effectively managing connection persistence. This lack of control resulted in inconsistent connection behaviors, where long-lived sessions might be terminated unexpectedly or held open longer than normal. With this release, the `HTTPKeepAliveTimeout` tuning option has been integrated into the `IngressController` API, providing a formal way for customers to configure and enforce this specific timeout. As a result, cluster administrators now possess the granular control necessary to align connection persistence with specific application needs. (link:https://issues.redhat.com/browse/OCPBUGS-68378[OCPBUGS-68378])

Reviewer: ```suggestion
* Before this update, {product-title} 4.16 and later versions failed to respect the timeout `http-keep-alive` setting due to a known upstream High HAProxy bug, preventing users from effectively managing connection persistence. This lack of control resulted in inconsistent connection behaviors, where long-lived sessions might be terminated unexpectedly or held open longer than normal. With this release, the `HTTPKeepAliveTimeout` tuning option has been integrated into the `IngressController` API, providing a formal way for customers to configure and enforce this specific timeout. As a result, cluster administrators now possess the granular control necessary to align connection persistence with specific application needs. (link:https://issues.redhat.com/browse/OCPBUGS-68378[OCPBUGS-68378])
```

* Before this update, the *Operand* details page would incorrectly show information using only half of the screen's display area in the OpenShift web console. With this release, the *Operand* details take up the full page width as expected.  (link:https://issues.redhat.com/browse/OCPBUGS-69677[OCPBUGS-69677])

> * Before this update, iptables-alerter pods experienced high CPU usage in some clusters due to an issue in the 4.18.20 upgrade. As a consequence, high CPU usage impacted iptables-alerter pods, causing performance degradation. With this release, iptables-alerter CPU usage has been reduced by optimizing code in version 4.18.21. As a result, high CPU usage for iptables-alerter pods has been resolved, improving cluster performance. (link:https://issues.redhat.com/browse/OCPBUGS-70328[OCPBUGS-70328])

Reviewer: Do we need backticks here? 
```suggestion
* Before this update, `iptables-alerter` pods experienced high CPU usage in some clusters due to an issue in the 4.18.20 upgrade. As a consequence, high CPU usage impacted `iptables-alerter` pods, causing performance degradation. With this release, `iptables-alerter` pod CPU usage has been reduced by optimizing code in version 4.18.21. As a result, high CPU usage for `iptables-alerter` pods has been resolved, improving cluster performance. (link:https://issues.redhat.com/browse/OCPBUGS-70328[OCPBUGS-70328])
```
